### DISCUSSION TITLE
How can i change this tab color

### DISCUSSION THREAD

**Discussion by ravisrma** [Godiesc/firefox-one#21] [General]
Can you help me out how i will change the color of the tabs
![Screenshot 2023-12-25 121110](https://github.com/Godiesc/firefox-one/assets/67187238/054272ab-2e86-4e86-af28-7dd1eb6ec7a0)

**Comment by ravisrma** [comment]
seprator clr

**Comment by Godiesc** [comment]
I don't understand it. Those are container tabs, you want change for example the red color with other less intense or what?
